tests/functional/claim_with_data.ts

```diff
@@ -0,0 +1,132 @@
+import { Claim, ClaimType, createClaim } from '@po.et/poet-js'
+const url = require('url')
+import fetch from 'node-fetch'
+import { describe } from 'riteway'
+const ipfsAPI = require('ipfs-api')
+
+const ipfsUrl = () => {
+  const { hostname, port } = url.parse(process.env.IPFS_URL)
+  return [hostname, port]
+}
+
+const ipfs = ipfsAPI(ipfsUrl()[0], ipfsUrl()[1], { protocol: 'http' })
+
+const getIPFSFileContents = async (hash: string) => {
+  const file = await ipfs.files.cat(hash)
+  return JSON.parse(file.toString('utf8'))
+}
+
+const privateKey = 'L1mptZyB6aWkiJU7dvAK4UUjLSaqzcRNYJn3KuAA7oEVyiNn3ZPF'
+const publicKey = '02cab54b227f16dd4866310799842cdd239f2adb56d0a3789519c6f43a892a61f6'
+
+// TODO: change default to NODE_A_URL
+const fullUrl = (host: string = process.env.INTEGRATION_TEST_NODE_URL) => (rest: any) => `${host}${rest}`
+
+const postWork = (claim: Claim) => {
```

Comment:
This was already implemented in the Integration tests, which I named incorrectly since they're actually functional tests :disappointed:

Reply:
While the integration tests informed the code used here, sharing code does not seem appropriate given that these are  tests from an end-user perspective and all code should be explicitly included here. They serve as a guide/documentation for anyone wishing to talk directly to a poet node.